climate-monitoring-core/src/main/java/climatemonitoring/core/View.java

```diff
@@ -1,6 +1,46 @@
+/*
+
+Alessandro della Frattina 753073 VA
+Cristian Capiferri 752918 VA
+Francesco Lops 753175 VA
+Dariia Sniezhko 753057 VA
+
+*/
+
 package climatemonitoring.core;
 
-public class View {
+/**
+ * The View class manages the state of a view and handles rendering operations based on the configuration (headless or GUI)
+ * @author francescolops
+ * @version 1.0-SNAPSHOT
+ */
+public class View{
 
+	/**
+	 * Sets the view state
+	 * @param state
```

Comment:
You could add a description that says: `The desired state` or else just remove the `@param` tag